dist/templates/k8s.ovn.org_nodednsinfos.yaml.j2

```diff
@@ -0,0 +1,74 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: nodednsinfoes.k8s.ovn.org
+spec:
+  group: k8s.ovn.org
+  names:
+    kind: NodeDNSInfo
+    listKind: NodeDNSInfoList
+    plural: nodednsinfoes
```

Comment:
NodeDNSInfos?

Reply:
that was an issue with code generation that I figured out